scripts/jest/jest.config.js

```diff
@@ -1,26 +1,31 @@
+const ignores = ['node_modules', '__fixtures__', '__mocks__', 'dist', 'lib'];
+
 module.exports = {
     clearMocks: true,
-    restoreMocks: true,
+    collectCoverage: false,
+    collectCoverageFrom: ['src/**/*.ts', 'src/**/*.tsx', 'src/**/*.js', '!**/node_modules/**', '!**/__tests__/**'],
+    coverageDirectory: '<rootDir>/reports',
+    coveragePathIgnorePatterns: ['\\.stories.*$', 'src/icon/*', 'src/icons/*', 'src/illustration'],
+    globalSetup: '<rootDir>/scripts/jest/env-setup.js',
     moduleNameMapper: {
         'box-ui-elements-locale-data': '<rootDir>/i18n/en-US.js',
         'box-locale-data': '<rootDir>/node_modules/@box/cldr-data/locale-data/en-US',
         '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
             '<rootDir>/scripts/jest/mocks/fileMock.js',
         '\\.(css|less|scss|md)$': '<rootDir>/scripts/jest/mocks/styleMock.js',
+        '^uuid$': require.resolve('uuid'),
     },
-    transformIgnorePatterns: ['node_modules/(?!(@box/react-virtualized/dist/es|@box/cldr-data))'],
-    testPathIgnorePatterns: ['/node_modules/', 'stories.test.js$', 'stories.test.tsx$'],
-    collectCoverage: false,
-    coverageDirectory: '<rootDir>/reports',
-    collectCoverageFrom: ['src/**/*.ts', 'src/**/*.tsx', 'src/**/*.js', '!**/node_modules/**', '!**/__tests__/**'],
-    coveragePathIgnorePatterns: ['\\.stories.*$', 'src/icon/*', 'src/icons/*', 'src/illustration'],
-    globalSetup: '<rootDir>/scripts/jest/env-setup.js',
-    roots: ['src'],
+    restoreMocks: true,
     rootDir: '../../',
+    roots: ['src'],
     setupFilesAfterEnv: [
         '<rootDir>/node_modules/regenerator-runtime/runtime.js',
         '<rootDir>/scripts/jest/enzyme-adapter.js',
         '<rootDir>/scripts/jest/jest-setup.js',
     ],
     snapshotSerializers: ['enzyme-to-json/serializer'],
+    testEnvironment: 'jsdom',
+    testMatch: ['**/__tests__/**/*.test.+(js|jsx|ts|tsx)'],
+    testPathIgnorePatterns: [...ignores, 'stories.test.js$', 'stories.test.tsx$', 'stories.test.d.ts'],
```

Comment:
is `ignores` only used here? does it need to be extracted into a separate variable?